helper-scripts/options/balenaio.json

```diff
@@ -1,81 +1,59 @@
 {
   "modelName": "balenaio",
-  "numConsecutiveDays": 5,
-  "supportStartHour": 6,
-  "supportEndHour": 27,
-  "slotsInDay": 27,
+  "numDays": 5,
+  "startHour": 6,
+  "endHour": 27,
+  "hoursInDay": 27,
```

Comment:
Shouldn't `hoursInDay` be `21` if `hoursInDay  = endHour - startHour`?

Reply:
Come to think of it, why do we need to specify the `hoursInDay`? Does this also have to do with the number of tracks in some hours of the day?

Reply:
@cmfcruz , you're right, there's no reason anymore to specify the `hoursInDay` separately. I've removed it from the code completely, since all the related functionality could be expressed in terms of `endHour` instead.